bower.json

```diff
@@ -14,6 +14,6 @@
     "rsvp": "https://github.com/tildeio/rsvp.js.git#3.0.20",
     "router.js": "https://github.com/tildeio/router.js.git#ed45bc5c5e055af0ab875ef2c52feda792ee23e4",
     "dag-map": "https://github.com/krisselden/dag-map.git#e307363256fe918f426e5a646cb5f5062d3245be",
-    "ember-dev": "https://github.com/emberjs/ember-dev.git#a064f0cd2f4c225ffd023b63d4cb31a79db04aaf"
+    "ember-dev": "mmun/ember-dev.git#modulized-debug-funcs"
```

Comment:
I think this should be `mmun/ember-dev#modulized-debug-funcs` (but not a biggie since we will land the fixes in ember-dev first).